doc/Programming-guide/Key-platform-features/Data-collection/MongoDB-log-appender/index.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: page
+title: MongoDB log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/MongoDB-log-appender
+sort_idx: 70
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The MongoDB log appender is responsible for transferring logs from the Operations service to the MongoDB database. The logs are stored in the table named
+__logs_$[applicationToken]({{root_url}}Glossary)_, where _$[applicationToken]({{root_url}}Glossary)_ matches the token of the current application.
+
+# Creating MongoDB log appender in Admin UI
+
+The easiest way to create a MongoDB log appender for your application is by using Admin UI.
+
+To create a log appender of the MongoDB storage type, do the following:
+
+1. In the **Log appenders** window, click **Add log appender**.
+2. Enter the log appender name and description, select the minimum and maximum supported log schema version, and select necessary log metadata fields.
+3. Set the log appender type to _Mongo_.
+4. Fill in the Mongo log appender configuration form.
+5. Click **Add** button. Log appender is ready and operational at this point.
+
+![Add log appender in Admin UI](attach/add-log-appender-in-admin-ui.png)
+
+# Creating MongoDB log appender with Admin REST API
+
+It is also possible to create a MongoDB log appender for your application by using [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs #TODO).
+The following example illustrates how to provision the MongoDB log appender for the Cell Monitor demo application available in Kaa Sandbox.
+
+## Configuration
+
+The MongoDB log appender configuration must match to
+ [this](https://github.com/kaaproject/kaa/blob/master/server/appenders/mongo-appender/src/main/avro/mongodb-appender-config.avsc) Avro schema.
+
+Fields of avro schema:
+
+* MongoDB nodes - list of MongoDB hosts.
+* Authentication credentials - credentials used to authenticate on MongoDB cluster
+* Other fields which configure of connection to MongoDB
+
+|name|description|
+|---|---|
+|dbName|name of database|
+|connectionsPerHost|Max number of connections on per host|
+|maxWaitTime|Max wait time for connection(ms)|
+|connectionTimeout|Connection timeout (ms)|
+|socketTimeout|Socket timeout (ms)|
+|socketKeepalive|Turn on socket keep alive (boolean value)|
+|includeClientProfile|Include or not client-side endpoint profile data (boolean value)|
```

Comment:
~~Include or not~~ Whether to include client-side endpoint profile data (boolean value)